build/cqs.tsv

```diff
@@ -73,9 +73,9 @@ de.topobyte:osm4j-utils	0.0.22	compile
 de.topobyte:osm4j-xml	0.0.3	compile
 eu.medsea.mimeutil:mime-util	2.1.3	compile
 io.airlift:aircompressor	0.10	compile
-io.dropwizard.metrics:metrics-core	3.1.2	compile
-io.dropwizard.metrics:metrics-ganglia	3.1.2	compile
-io.dropwizard.metrics:metrics-graphite	3.1.2	compile
+io.dropwizard.metrics:metrics-core	3.2.6	compile
+io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
+io.dropwizard.metrics:metrics-graphite	3.2.6	compile
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20220

Reply:
@jnh5y could you ping the pmc for this one? Unless you want to wait and see how/if spring shakes out.

Reply:
@elahrvivaz can we get a PR up for Spring first so that I can ask for all the CQs as once?

Reply:
@jnh5y I opened a PR for spring here https://github.com/locationtech/geomesa/pull/2300, but it only requires piggy-back cqs so they should be automatically approved.